### PR TITLE
Build visualization

### PR DESCRIPTION
**Pull request by mhyder1**
# Proposed Changes

Describe the big picture of your changes here.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

- [ ] My changes generate no new warnings
- [ ] Logging has been added where applicable (Error handling, user-driven events, API requests/responses)

- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] Tests meet the test coverage threshold

## Reviewer Checklist (rianfowler)

- [x] Is the code easy to understand?
- [ ] Are the changes well documented?

- [x] Can you think of any use case in which the code does not behave as intended? No
- [x] Can you think of any inputs or external events that could break the code? No

- [x] Should any logging or debugging information be added or removed? no

## Further Comments

If this is a relatively large or complex change, kick off the discussion by explaining how you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by keyluck — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -58,7 +59,8 @@
     ]
   },
   "dependencies": {
-    "@changesets/cli": "^2.16.0"
+    "@changesets/cli": "^2.16.0",
+    "node-gyp": "^8.3.0"
```
Comment (rianfowler) [inline package.json:63]: What is this used for?

File: packages/app/src/App.tsx [hunk 2]
```diff
@@ -38,6 +38,10 @@ import { lightThemeVA, darkThemeVA } from './themes/index';
 import { FeatureFlagsPage } from '@internal/plugin-feature-flags';
 import { FeatureFlagRegistry } from './FeatureFLagRegistry';
 
+const Datadog = () => (
+  <iframe style={{width:'100vw', height:'100vh'}} title="dashboard" src="https://p.datadoghq.com/sb/dq828b4kr813cbhm-127a5292a5c6656e73d7ab859c3b46a3"/>
+)
+
```
Comment (rianfowler) [inline packages/app/src/App.tsx:44]: What do you think about adding this to the components folder like the `homepage` and importing it? I know it's pretty small but we may want to keep component definitions out of the `App.tsx`

**Review by rianfowler — APPROVED** [review]
This looks good. If you'd like I think we could spend one of the working sessions this week talking through how we'd move this into the catalog entry page and how we might make that configurable for other entries in backstage.

File: package.json (hunk 1 above)
Comment (mhyder1) [reply package.json:63]: I'm not sure why changesets are there. I didn't purposely add it. I can double check the history to see when it was added

File: packages/app/src/App.tsx (hunk 2 above)
Comment (mhyder1) [reply packages/app/src/App.tsx:44]: Sounds like a good idea.

File: package.json (hunk 1 above)
Comment (mhyder1) [reply package.json:63]: I remember now. I was trying to get act to work properly. I would up needing to just use the later docker image. Removing...

**Review by KaemonIsland — APPROVED** [review]

**Pull request merged**